docs/development.rst

```diff
@@ -90,32 +90,36 @@ Tablib features a micro-framework for adding format support.
 The easiest way to understand it is to use it.
 So, let's define our own format, named *xxx*.
 
-1. Write a new format interface.
+From version 0.15.0, Tablib formats are class-based and can be dynamically
```

Comment:
Will the next one be 1.0, because Python 2 has been dropped?

Re: https://github.com/jazzband/tablib/issues/390#issuecomment-538331435